lhctl/cmd/node_run.go

```diff
@@ -62,22 +62,43 @@ var getNodeRunCmd = &cobra.Command{
 }
 
 var listNodeRunCmd = &cobra.Command{
-	Use:   "nodeRun <wfRunId>",
+	Use:   "nodeRun <wfRunId> <threadRunNumber>",
 	Short: "List all NodeRun's for a given WfRun Id.",
+	Args: func(cmd *cobra.Command, args []string) error {
+		if err := cobra.MinimumNArgs(1)(cmd, args); err != nil {
+			return err
+		}
+
+		if err := cobra.MaximumNArgs(2)(cmd, args); err != nil {
+			return err
+		}
+
+		return nil
```

Comment:
This handles the argument count, so there's no need to handle in the `Run` function.

Reply:
Much better. Thanks